app/views/stream_entries/_og_description.html.haml

```diff
@@ -1,4 +1 @@
-- if activity.is_a?(Status) && activity.spoiler_text?
-  = opengraph 'og:description', activity.spoiler_text
-- else
-  = opengraph 'og:description', activity.content
+= opengraph 'og:description', [activity.spoiler_text, activity.text].reject(&:blank?).join("\n\n")
```

Comment:
should maybe be `"CW: " + activity.spoiler_text`?

Reply:
I know we have "CW" on a button in the UI but adding it inside of a user's post could be problematic because it's an English string.

Reply:
(late comment I know, but...)
I tried this just now, and in addition to being a hardcoded English string it also ends up prepending a "CW: " line on top of non-CWed toots as well. But it would be a good idea to try and add this so that, say if I'm linking toots in Discord with people who don't use masto, they have the context that this line up top is a content warning without me having to say so or having to click through.